src/content/smart-bulletins/031.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Smart Bulletin No. 031"
+intro: "GSA SmartPay – OMB Circular A-123, Appendix B Implementing Guidance and Clarifications/Updates"
+number: "No. 031"
+order: 031
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Dec 19, 2019"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| December 19, 2019 | Implementation of Smart Bulletin #31 |
+
+## Effective Date
+
+Effective upon issuance and shall remain in force until modified,
+rescinded, or superseded.
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide implementation guidance, updates and clarifications relating to portions of OMB Circular A-123, Appendix B, ‘A Risk Framework for Charge Card Programs (“the Circular”). These revisions were coordinated with the Office of Management and Budget, Office of Federal Financial Management.
```

Comment:
The purpose of this GSA SmartPay Smart Bulletin is to provide implementation guidance, updates and clarifications relating to portions of OMB Circular A-123, Appendix B, ‘A Risk Framework for Charge Card Programs (“the Circular”). These revisions were coordinated with the Office of Management and Budget Office of Federal Financial Management.